src/komodo_validation011.h

```diff
@@ -748,15 +754,14 @@ void komodo_importpubkeys()
         if ( (m= (int32_t)strlen((char *)Notaries_elected1[i][offset])) > 0 )
         {
 	    pubkey = (char*) Notaries_elected1[i][offset];
-	    //fprintf(stderr,"pubkey=%s\n", pubkey );
 
 	    const std::vector<unsigned char> vPubkey(pubkey, pubkey + m);
 	    std::string addr = CBitcoinAddress(CPubKey(ParseHex(pubkey)).GetID()).ToString();
 
-	    //fprintf(stderr,"addr=%s\n", addr.c_str() );
+	    //fprintf(stderr,"pubkey=%s, addr=%s\n", pubkey, addr.c_str() );
 
             if ( (val= komodo_importaddress(addr)) < 0 )
-                fprintf(stderr,"error importing (%s)\n",addr.c_str());
+                LogPrintf("dpow: error importing (%s)\n",addr.c_str());
```

Comment:
Could use `LogPrint("dpow", "error importing (%s)\n",addr.c_str());` so that those logs can be toggled off individually.